driver/bpf/filler_helpers.h

```diff
@@ -357,6 +357,14 @@ static __always_inline u32 bpf_compute_snaplen(struct filler_data *data,
 		return 2000;
 	} else if (dport == PPM_PORT_STATSD) {
 		return 2000;
+	} else if (data->settings->fullcapture_port_range_end != 0 &&
+				((sport >= data->settings->fullcapture_port_range_start && sport <= data->settings->fullcapture_port_range_end) ||
+				(dport >= data->settings->fullcapture_port_range_start && dport <= data->settings->fullcapture_port_range_end)
+			)) {
+		/*
+		* mpegts detection
+		*/
+		return 16000;
```

Comment:
Minor: eventually move this to a define since it's shared between bpf and driver (wasn't done for the other part of the function as part of the initial porting because I didn't want to cause a large diff in the driver code). Not a deal breaker considering the current status.